roland/Layouts/LayoutRSS.swift

```diff
@@ -27,7 +27,7 @@ class LayoutRSS: Layout {
             return nil
         }
 
-        template = Template(templateName: "RSS", website: website)
+        template = Template(templateName: "rss", website: website)
```

Comment:
Was there a reason for lowercasing `rss`?

Reply:
mac is case-insensitve, but Linux isn't.